src/libraries/System.Text.RegularExpressions/src/ILLink/ILLink.Descriptors.LibraryBuild.xml

```diff
@@ -0,0 +1,10 @@
+<linker>
+  <assembly fullname="System.Text.RegularExpressions">
+    <type fullname="System.Text.RegularExpressions.Regex">
+      <!-- called via reflection from System.Text.RegularExpressions.Tests -->
+      <method name="Serialize" />
```

Comment:
It would be triggered by using reflection, same as in the unit tests:
MethodInfo _Deserialize = typeof(Regex).GetMethod("Deserialize", BindingFlags.NonPublic | BindingFlags.Static);
Regex Deserialize(string s) => _Deserialize.Invoke(null, new object[] { s }) as Regex;

MethodInfo _Serialize = typeof(Regex).GetMethod("Serialize", BindingFlags.NonPublic | BindingFlags.Instance);
string Serialize(Regex r) => _Serialize.Invoke(r, null) as string;
For example, CredScan would be ok with that, I asked specifically if using reflection would be fine for them.